### DISCUSSION TITLE
Database / Backend

### DISCUSSION THREAD

**Discussion by allanoguis** [allanoguis/Hacktoberfest-2024#49] [Ideas]
About the issue #48 : the decision is currently MongoDB / Firebase for data storage

**Comment by Parth-G27** [comment]
Hi, so Like I suggested
I think this can be the schema for the score table for multiplayer

{
_id: default
score: int
time: Datetime
}

we can go with MongoDB or Firebase NoSQL, both will work.

**Comment by vaibhav-gosavi** [comment]
lets go with MongoDB

**Comment by allanoguis** [comment]
@Parth-G27  We can scale the tables in case we need other data like emails, names, etc in future yea?

**Reply by vaibhav-gosavi** [reply]
yes

**Reply by allanoguis** [reply]
Cool! Alright I'll leave you guys to it. I can't make a PR yet there are still bugs in the game but I was able to make changes to it. the jump and collision / possible hitboxes are the problem

**Reply by Parth-G27** [reply]
yes we can scale the table with other columns also

**Reply by vaibhav-gosavi** [reply]
ok go ahead
